pages/linux/rpcclient.md

```diff
@@ -0,0 +1,36 @@
+# rpcclient
+
+> MS-RPC client tool (part of the samba suite).
+> More information: <https://www.samba.org/samba/docs/current/man-html/rpcclient.1.html>.
+
+- Connect to a host:
+
+`rpcclient --user {{domain\username%password}} {{ip}}`
+
+- Connect to a host on a domain without a password:
+
+`rpcclient --user {{username}} --workgroup {{domain}} --no-pass {{ip}}`
+
+- Connect to a host, passing the password hash:
+
+`rpcclient --user {{domain\username}} --pw-nt-hash {{ip}}`
+
+- Enumerate domain users:
+
+`rpcclient $> enumdomusers`
```

Comment:
Based on `vim.md` we don't include the prompt:
```suggestion
`enumdomusers`
```

Reply:
Cfr [`evil-winrm.md`](https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/evil-winrm.md), I think it's better to state the context as these commands are only valid in the scope of the sub-terminal once connected using the tool.
What is your opinion ?